Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -368,6 +376,112 @@ extension BrowserViewController: BrowserToolbarDelegate {
     }
 }
 
+private var bars = [SnackBar]()
```

Comment:
Why do we need two copies  of the `[SnackBar]` array? One in the `BrowserViewController` and one in `Browse`. This, combined with `realRemoveBar` and `realAddBar` don't give me a good feeling about this code.